types/src/block_metadata.rs

```diff
@@ -69,7 +69,7 @@ impl BlockMetadata {
 }
 
 pub fn new_block_event_key() -> EventKey {
-    EventKey::new_from_address(&libra_root_address(), 20)
+    EventKey::new_from_address(&libra_root_address(), 21)
```

Comment:
This is an unforunate consequnce of this change as we changed the order of how event handles are generated. Do we want to keep the same magic value?

Reply:
Update: I got rid of this change by reorganizing the genesis intialization code, as this will also change the event key for a number of other system events (all mint related events).